models.rdf

```diff
@@ -66,6 +66,14 @@
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdfs:Class>
 
+    <rdfs:Class rdf:about="http://pcdm.org/models#FileSet">
```

Comment:
What is FileSet a subClassOf ? directly to ore:Aggregation, or off of pcdm:Object?